native/dgebrd.go

```diff
@@ -0,0 +1,92 @@
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dgebrd reduces a general m×n matrix A to upper or lower bidiagonal form B by
+// an orthogonal transformation: Q^T * A * P = B.
+func (impl Implementation) Dgebrd(m, n int, a []float64, lda int, d, e, tauQ, tauP, work []float64, lwork int) {
```

Comment:
Unfortunately, the layout is not the same as Dlabrd so it needs its own documentation.